examples/cmp-vendors.amp.html

```diff
@@ -0,0 +1,102 @@
+    }
+    select {
+      border: 1px solid #999;
+    }
+    input[type=submit] {
+      font-size: 14px;
+      border: 1px solid #999;
+      border-radius: 3px;
+      margin-left: 10px;
+    }
+    </style>
+    <script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>
+    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+    <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
+  </head>
+  <body>
+
+    <div class="filterBar">
+      <form action="cmp-vendors.amp.html" method="get" target="_top">
+        <label for="filter">Vendor filter</label>
+        <select id="filter" name="type">
+          <option>[Select a vendor]</option>
+          <!-- This should be in alphabetical order. _ping_ should be the first -->
+          <option>_ping_</option>
+        </select>
+        <input type="submit" value="Go">
+      </form>
+    </div>
+
+    <h1 id="top">AMP Consent CMP</h1>
+    <h3>Image that is blocked by both consent</h3>
+    <amp-img data-block-on-consent src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w300-h200-no-n" width=300 height=200></amp-img>
+
+    <h3>Image that is NOT blocked by consent</h3>
+    <amp-img src="https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w300-h200-no-n" width=300 height=200></amp-img>
+
+
+    <div id="container">
+      Container for amp-consent tags.
+
+      <!-- This should be in alphabetical order. _ping_ should be the first -->
+
+      <!-- _ping_ example -->
+      <amp-consent id='ABC' layout='nodisplay' type='_ping_'>
+        <script type="application/json">
+          {
+            "consents": {},
+              "postPromptUI": "postPromptUI",
+              "clientConfig": {
+                "CMP_id": "test_id",
+                "other_info": "test_info"
+              }
+          }
+        </script>
+        <div id="ui1">
```

Comment:
nit: 'ui1' and 'ui2' not used, can be removed

Reply:
Yes it is? I wanted prompts so vendors can see their UI working:

<img width="667" alt="Screen Shot 2019-05-08 at 5 53 48 PM" src="https://user-images.githubusercontent.com/1448289/57420058-6ee70380-71ba-11e9-909f-8a56f00f08ff.png">

I think the confusion may have been from the experiment not being toggled? So I added a note as well. And updated the original PR screenshot 😄

Reply:
I believe the UI is from the _ping_ iframe, where you specify using `'promptUISrc': '/test/manual/diy-consent.html',`

I don't think the `ui1` and `ui2` are used as they are not referred anywhere.

Reply:
Oh duh 😂 Thank you!